pkg/auth/cookie_manager.go

```diff
@@ -20,6 +20,8 @@ const (
 	ErrB64Decoding errors.ErrorCode = "BINARY_DECODING_FAILED"
 	// #nosec
 	ErrTokenNil errors.ErrorCode = "EMPTY_OAUTH_TOKEN"
+	// #nosec
```

Comment:
huh I wonder do we still need this? can't remember if it was lyft security scanning that required this originally

Reply:
Apparently we still do... golangci-lint complains when I remove them...